src/features/search/helpers/categoriesHelpers/categoriesHelpers.ts

```diff
@@ -62,14 +62,24 @@ export function getSearchGroupsFromEnumArray(
     .filter(Boolean) as SearchGroupResponseModelv2[]
 }
 
+/**
+ * Returns a `NativeCategoryResponseModelv2` value from a `NativeCategoryIdEnumv2`.
+ */
+export function getNativeCategoryValue(
+  data: SubcategoriesResponseModelv2 | undefined,
+  enumValue: NativeCategoryIdEnumv2
+) {
+  return data?.nativeCategories.find((nativeCategory) => nativeCategory.name === enumValue)?.value
+}
```

Comment:
Tu dupliques inutilement du code.
Tu pourrais faire 
```suggestion
export function getNativeCategoryValue(
  data: SubcategoriesResponseModelv2 | undefined,
  enumValue: NativeCategoryIdEnumv2
) {
  return getNativeCategoryFromEnum(data, enumValue)?.value
}
```

Reply:
Par ailleurs, tu n'utilises cette fonction qu'une seule fois et disons-le, elle n'est pas très utile.
Tu pourrais simplement, à l'endroit où tu l'utilises aujourd'hui, faire un 
```ts
const { value } = getNativeCategory(data, nativeCategoryId) || {}
```
et supprimer cet helper.

Ça ne complexifie pas tellement le code, et ça évite un énième helper.

Reply:
Comme tu veux tu choisis :p